radlab-ui/webapp/src/routes/Deployments.tsx

```diff
@@ -134,28 +180,54 @@ const Deployments: React.FC<DeploymentsProps> = () => {
           )}
         </div>
       </div>
-      {deploymentTab === DEPLOYMENT_TAB.ALL &&
-        (listAllDeployments?.length ? (
-          <ModuleDeployment
-            headers={DEPLOYMENT_HEADERS}
-            deployments={listAllDeployments}
-            defaultSortField={SORT_FIELD.CREATEDAT}
-            defaultSortDirection={SORT_DIRECTION.DESC}
-          />
-        ) : (
-          renderEmptyState()
-        ))}
-      {deploymentTab === DEPLOYMENT_TAB.MINE &&
-        (listMyDeployments?.length ? (
-          <ModuleDeployment
-            headers={DEPLOYMENT_HEADERS}
-            deployments={listMyDeployments}
-            defaultSortField={SORT_FIELD.CREATEDAT}
-            defaultSortDirection={SORT_DIRECTION.DESC}
-          />
-        ) : (
-          renderEmptyState()
-        ))}
+      <div className="bg-base-100 rounded-md shadow-md">
+        <div className="w-full p-4">
+          {!isLoading && (
+            <Filter
+              filters={["module", "createdAt", "status"]}
+              // @ts-ignore
+              statuses={activeStatuses}
+              modules={modules}
+            />
+          )}
+        </div>
+      </div>
+      {deploymentTab === DEPLOYMENT_TAB.ALL && (
+        <>
+          {listAllDeployments?.length ? (
```

Comment:
You don't need this first check

Reply:
`renderEmptySearch()` is a function which will be used when we search and find that no data is present according to search and `renderEmptyState` is used when there is no deployment.

Reply:
Should I write some other logic to handle the condition?